src/app/containers/MediaPlayer/helpers/metadata/index.js

```diff
@@ -1,5 +1,12 @@
 import pathOr from 'ramda/src/pathOr';
 
+const DEFAULT_IMAGE_RES = '640x360';
```

Comment:
Where does this default value originate from?

Reply:
@jamesdonoh I saw something similar [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Image/index.jsx#L14), the associated `aresmetadata` block doesn't specify Image resolution (height and width)

Reply:
I think we should verify with UX before putting these values in.

Reply:
OK thanks for explanation, do not think it needs a UX review at this point. But what happens if the image has a different aspect ratio? Does it work if we just specify the width? (Looking at https://github.com/bbc/simorgh/blob/latest/src/app/containers/Image/index.jsx#L29)

Reply:
@jamesdonoh @12  my understanding of this is that the essence of specifying the image resolution is to ascertain the best quality image that will show up on search engines, so I think we would have to just agree on what image sizes we prefer and I think UX would be a good place to start. The `image` block within `aresmedia` block has a `height` and `width` property but those always have the largest values (1920x1080) and I don't think we could use those as well

Reply:
I'd rather put something in that renders an image at all and then improve the quality/sizes with UX later on - the only critical thing is that the aspect ratio is preserved. Have you checked whether imagechef allows us to just specify the code I linked to above appears to be doing? If so we can just specify a width of 640 and that will do for now.

Reply:
@jamesdonoh I tried using just the width in this case but that doesn't resolve to any ichef image

Reply:
I think it's worth checking with UX and our SEO specialists. The 640x360 default const is what we are willing to display to users. Whereas our metadata images tend to be much bigger as the likes of Twitter, Facebook and Google will compress the image. For PAL the Open Graph images are 1024px width because of this reason, I'd suggest using `1024x576` here (EG: http://ichef.bbci.co.uk/images/ic/1024x576/p07f588v.jpg) 

Also worth mentioning that we have to provide a height and width to ichef (rather than just width as James mentioned above) because these images come from Nitro and only respond to [certain ratio's](https://github.com/BBC-News/responsive-news/blob/develop/tabloid/webapp/static/js/module/imageenhancer.js#L18-L21).

Reply:
Thanks for the explanation Phil, @rhenshaw56 please can you use 1024x576 for now, we can arrange for SEO colleagues to take a look once it's on test as part of a general review of AV SEO.